frontend/i18n/locales/de.json

```
@@ -173,5 +175,47 @@
   },
   "infobox": {
     "current_of_total": "{current} von {total}"
+  },
+  "homepage": {
+    "project_description": "Lerne die agile Projektmanagement\u00ADmethode Scrum mittels Bau einer Papierstadt! ScrumpliCity stellt eine Web-Applikation, Bastelvorlagen und Erklärungen zu Scrum zur Verfügung - also: ",
+    "build_your_scrum_knowledge": "Build Your Scrum Knowledge",
+    "get_started": "Loslegen!",
+    "scrum_guide": "Scrum Guide",
+    "diploma_thesis_website": "Diplomarbeits-Webseite",
+    "crafting_templates_download": "Bastelvorlagen-Download",
+    "follow_us": "Folge uns!",
+    "about_us": "Über uns",
+    "features": {
+      "learn_scrum": {
+        "title": "Lerne Scrum",
+        "description": "Entdecke die Grundlagen von Scrum und starte dein erstes agiles Projekt — praxisnah, interaktiv und einfach verständlich!"
+      },
+      "craft_your_city": {
+        "title": "Bastle deine Stadt",
+        "description": "Gestalte deine eigene Stadt aus Papier und erwecke Schritt für Schritt dein Projekt zum Leben!"
+      },
+      "solidify_teamwork": {
+        "title": "Festige Teamwork",
+        "description": "Stärke den Teamgeist und erlebe, wie effektive, kooperative Zusammenarbeit deine Projekte zum Erfolg führen!"
+      }
+    },
+    "learning_materials": "Lernunterlagen",
+    "all_you_need_to_know": "Scrum: Alles was du wissen musst!",
+    "scrum_guide_description": "Du möchtest dich außerhalb des Spielprozesses über die agile Projektmanagement&shy;methode Scrum weiterbilden? Auf unserer Scrum-Theorie-Seite findest du alles Wichtige: von den Grundlagen bis zu praktischen Tipps. Egal, ob du neu einsteigst oder dein Wissen vertiefen möchtest — hier bekommst du klare, verständliche Erklärungen. Schau vorbei und bring deine Projekte auf das nächste Level!",
```

```suggestion
    "scrum_guide_description": "Du möchtest dich außerhalb des Spielprozesses über die agile Projektmanagement&shy;methode Scrum weiterbilden? Auf unserer Scrum-Theorie-Seite findest du alles Wichtige: von den Grundlagen bis zu praktischen Tipps. Egal, ob du neu einsteigst oder dein Wissen vertiefen möchtest – hier bekommst du klare, verständliche Erklärungen. Schau vorbei und bring deine Projekte auf das nächste Level!",
```
Gedankenstriche im Deutschen sind Halbgeviertstriche